Documentation/devel/on-disk-format.md

```diff
@@ -28,4 +28,3 @@ the CAS.
 
 The [configuration](https://github.com/coreos/rkt/blob/master/Documentation/configuration.md)
 on-disk format is documented separately.
-
```

Comment:
Actually the first sentence in this file is:
> The data directory is normally /var/lib/rkt but rkt can be requested to use another directory via the --dir command line argument.

Can this updated? Maybe something like:

> The data directory is normally /var/lib/rkt, unless configured otherwise. + link to configuration.md